client/src/components/Tickets/TicketCreateModal.scss

```diff
@@ -1,14 +1,6 @@
-.ticket-create-modal-body {
-  display: flex;
-  flex-direction: column;
-  overflow: auto;
-  padding-bottom: 1em;
-}
-
 .ticket-create-modal {
```

Comment:
Think this fixes the overflow issue
```suggestion
.ticket-create-modal-body {
  overflow: auto;
  max-height: 60vh;
}
.ticket-create-modal {
```

Reply:
@Strmiska does this need to be applied anywhere else?

Reply:
🤘 Thank you, @rstijerina.

_I secretly hoped there was a simple solution that I couldn't see. Yesterday, I was performing tasks with blinders on._

Reply:
This does fix the issue, and might be a better way to fix any other modals that may have this problem. I'm not sure which form are affected at the moment, but I'll start with committing these changes first.

Reply:
My `max-height` search found two other instances that may be relevant:
- https://github.com/TACC/Frontera-Portal/blob/master/client/src/components/Tickets/TicketModal.scss#L6-L9
- https://github.com/TACC/Frontera-Portal/blob/master/client/src/components/DataFiles/DataFilesModals/DataFilesPreviewModal.module.scss#L1-L3

Reply:
I think this issue only pops up when we're trying to render a form component from within a separate modal. I can't think of any other places where that exists or besides the Ticket modal...